ZionEscape/Game.h

```diff
@@ -24,6 +35,104 @@ ref class Game {
   int GetMapSeed() {
     return this->map->GetSeed();
   }
+
+  //MessageBox 
+
+  //Start the message box
+  void StartMessagebox() {
+    this->messagebox->SetActivated(true);
+  }
+
+  //Draw the Message
+  void DrawMessagebox(Graphics^ g) {
+    if (this->messagebox != nullptr && this->messagebox->GetActivated())
+        this->messagebox->Draw(g);
+  }
+  
+  //Set the message and len
+  void SetMessage(System::Windows::Forms::Label^ label) {
```

Comment:
Seguramente debes poder evitar poner el `namespace`. Pruébalo.

```suggestion
  void SetMessage(Label^ label) {
```